Maestro/subscriptions.json

```diff
@@ -1147,7 +1147,7 @@
     // Update dependencies in the aspnetcore repo's release/2.1 branch
     {
       "triggerPaths": [
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1.6/packages+installers.semaphore",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1.7/packages.semaphore",
```

Comment:
Are you planning to switch this back? Depending on both steps makes sure the installers are also ready when aspnet gets an upgrade PR. I can understand changing it so that you don't have to manually update the semaphore since the installers are already published though.

Reply:
Yeah, either that or I'll just delete the whole block